transport/grpc/dial.go

```diff
@@ -47,6 +49,26 @@ var logRateLimiter = rate.Sometimes{Interval: 1 * time.Second}
 // Assign to var for unit test replacement
 var dialContext = grpc.DialContext
 
+// otelGRPCClientHandler is a singleton otelgrpc clientHandler to be used across
+// all dial connections to avoid the memory leak documented in
+// https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4226
+//
+// TODO: If 4226 has been fixed in opentelemetry-go-contrib, replace this
+// singleton with inline usage for simplicity.
+var (
+	otelGRPCClientHandler stats.Handler
+	once                  sync.Once
```

Comment:
nit: just like mutexes with once it is considered better to put the field above the value it protects. Also since these are globals consider a name for the once that aligns it better to the thing it is protecting.

Reply:
Done, thanks!